apack.json

```diff
@@ -98,16 +99,16 @@
       "cmd": "$(zap) server --reuseZapInstance --studioHttpPort ${studioHttpPort}"
     },
     "zapFull": {
-      "cmd": "$(zap) --showUrl --zcl ${sdkRoot}/app/zcl/zcl-zap.json"
+      "cmd": "$(zap) --showUrl --zcl ${sdkRoot}/app/zcl/zcl-zap.json [ --zcl ${zcl.zigbeeZclJsonFile} ]"
     },
     "zapRegenSdk": {
       "cmd": "$(zap) regenerateSdk --sdk ${sdkJsonFile}"
     },
     "zapIscConvert": {
-      "cmd": "$(zap) convert --noUi --noServer --noZapFileLog --zcl ${sdkRoot}/app/zcl/zcl-zap.json --out ${outFile} ${inFile}"
+      "cmd": "$(zap) convert --noUi --noServer --noZapFileLog --zcl ${sdkRoot}/app/zcl/zcl-zap.json [ --zcl ${zcl.zigbeeZclJsonFile} ] --out ${outFile} ${inFile}"
     },
     "zapIscOpen": {
-      "cmd": "$(zap) --noServer --zcl ${sdkRoot}/app/zcl/zcl-zap.json --generationTemplate ${sdkRoot}/protocol/zigbee/app/framework/gen-template/gen-teplates.json ${inFile}"
+      "cmd": "$(zap) --noServer --zcl ${sdkRoot}/app/zcl/zcl-zap.json --generationTemplate ${sdkRoot}/protocol/zigbee/app/framework/gen-template/gen-templates.json [ --zcl ${zcl.zigbeeZclJsonFile} ] [ --generationTemplate ${zcl.zigbeeTemplateJsonFile} ] ${inFile}"
```

Comment:
Same thing. This makes little sense to me. This WILL fail if sdkRoot is not present...